recipes/octo-logger-cpp/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conans import ConanFile, CMake
+from conan.tools.files import get, collect_libs
+
+
+class OctoLoggerCPPConan(ConanFile):
+    name = "octo-logger-cpp"
+    version = "1.0.0"
+    license = "MIT"
+    url = "https://github.com/ofiriluz/octo-logger-cpp"
+    homepage = "https://github.com/ofiriluz/octo-logger-cpp"
+    description = "Octo logger library"
+    topics = ("logging", "cpp")
+    author = "Ofir Iluz"
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+
+    @property
+    def _source_subfolder(self):
+        return "source"
+
+    @property
+    def _build_subfolder(self):
+        return "build"
+
+    def source(self):
+        get(self, **self.conan_data["sources"][str(self.version)], strip_root=True)
+
+    def requirements(self):
+        self.requires("catch2/3.1.0")
+        self.requires("fmt/9.0.0")
+        self.requires("trompeloeil/42")
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder=self._source_subfolder, build_folder=self._build_subfolder)
+        cmake.build(build_dir=self._build_subfolder)
+        cmake.test(build_dir=self._build_subfolder)
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        cmake = CMake(self)
+        cmake.install(build_dir=self._build_subfolder)
+
+    def package_info(self):
+        self.cpp_info.libs = collect_libs(self)
```

Comment:
```suggestion
        self.cpp_info.libs = collect_libs(self)
```
Avoid collect_libs. It's fragile and may lie about the link order. Prefer declaring explicitly the library names.

Reply:
removed and added explicit info